independent-publisher-connectors/Google Photos/apiDefinition.swagger.json

```diff
@@ -0,0 +1,161 @@
+        "operationId": "CreateItems",
+        "parameters": [
+          {
+            "name": "mediaItemIds",
+            "default": "<string>",
+            "in": "query",
+            "type": "string",
+            "x-ms-summary": "Create Item",
+            "description": "Create Media Item - Photo, video etc",
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1/mediaItems:batchCreate": {},
+    "/v1/sharedAlbums": {
+      "get": {
+        "summary": "List All Shared Albums",
+        "description": "\n\nLists all shared albums available in the Sharing tab of the user's Google Photos app.\n\n### HTTP request\n\n`GET https://photoslibrary.googleapis.com/v1/sharedAlbums`\n\nThe URL uses [gRPC Transcoding](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) syntax.\n\n### Query parameters\n\n|Parameters|||\n|-|-|-\n|`pageSize`|`integer`|Maximum number of albums to return in the response. Fewer albums might be returned than the specified number. The default `pageSize` is 20, the maximum is 50.\n|`pageToken`|`string`|A continuation token to get the next page of the results. Adding this to the request returns the rows after the `pageToken`. The `pageToken` should be the value returned in the `nextPageToken` parameter in the response to the `listSharedAlbums` request.\n|`excludeNonAppCreatedData`|`boolean`|If set, the results exclude media items that were not created by this app. Defaults to false (all albums are returned). This field is ignored if the photoslibrary.readonly.appcreateddata scope is used.\n\n### Request body\n\nThe request body must be empty.\n\n### Response body\n\nIf successful, the response body contains data with the following structure:\n\nList of shared albums requested.\n\nJSON representation\n\n```json\n{\n  \"sharedAlbums\": [\n    {\n      object (Album)\n    }\n  ],\n  \"nextPageToken\": string\n}\n```\n\n|Fields|||\n|-|-|-\n|`sharedAlbums[]`|`object` ([Album](https://developers.google.com/photos/library/reference/rest/v1/albums#Album))|Output only. List of shared albums.\n|`nextPageToken`|`string`|Output only. Token to use to get the next set of shared albums. Populated if there are more shared albums to retrieve for this request.\n\n### Authorization Scopes\n\nRequires one of the following OAuth scopes:\n\n*   `https://www.googleapis.com/auth/photoslibrary`\n*   `https://www.googleapis.com/auth/photoslibrary.readonly`\n*   `https://www.googleapis.com/auth/photoslibrary.readonly.appcreateddata`\n",
+        "operationId": "ListSharedAlbums",
+        "parameters": [],
+        "responses": {
+          "default": {

```

Comment:
@juliamuiruri4 default responses is not allowed. Please define a proper response, please see [here](https://learn.microsoft.com/en-gb/connectors/custom-connectors/coding-standards#responses)